phoenix-core/src/it/java/org/apache/phoenix/end2end/index/PartialIndexIT.java

```diff
@@ -0,0 +1,479 @@
+/*
```

Comment:
Can you add a test case with atomic upserts and partial index. Specifically, when the row already exists and then after applying the atomic condition the column value changes and then the new column value satisfies the partial index condition.

Reply:
Yes, I should do that.

Reply:
Done